grails-app/views/projectActivity/_status.gsp

```diff
@@ -0,0 +1,13 @@
+<label>
```

Comment:
This should be a div or span, not a label: labels are for labelling input fields.